caom2/caom2/shape.py

```diff
@@ -350,6 +350,41 @@ def cval2(self, value):
 
 class Polygon(common.CaomObject):
 
+    def __init__(self, points=None, samples=None):
```

Comment:
Polygon requires one points and one samples element. Should they be allowed to be None? Then you have an empty polygon?